pkg/feature/api/feature.go

```diff
@@ -875,7 +875,7 @@ func (s *FeatureService) existsRunningProgressiveRollout(
 	ctx context.Context,
 	featureID, environmentNamespace string,
 ) (bool, error) {
-	progressiveRollouts, err := s.listProgressiveRollouts(ctx, environmentNamespace, featureID)
+	progressiveRollouts, err := s.listProgressiveRollouts(ctx, featureID, environmentNamespace)
```

Comment:
There was a bug here. I also added a unit test to ensure it's working.
I'm checking the values passed to the API.

https://github.com/bucketeer-io/bucketeer/pull/712/files#diff-fad41f54a190634386c37788c6f063b9d12ddb75b1928675e5b6ddd61b0d9663R3114-R3119